Core/MIPS/ARM64/Arm64CompLoadStore.cpp

```diff
@@ -366,7 +366,15 @@ namespace MIPSComp {
 			} else {
 				// This actually gets hit in micro machines! rs = ZR rt = ZR. Probably a bug.
 				// Leaving this a debug assert for future investigation.
+#ifdef __APPLE__
+				if (__builtin_expect(gpr.IsImm(rs), 0)) {
```

Comment:
__builtin_expect seems like a strange choice, how about a normal if?

_dbg_assert_msg_ should not be fatal on iOS release builds, it would be better to fix that instead.

Reply:
I used a debug build, it may indeed not crash on release. Will check later.

As for builtin, I did not want an extra penalty for an unfrequent if just in case.